### DISCUSSION TITLE
How to enable path excutable completion?

### DISCUSSION THREAD

**Discussion by NoahELE** [nushell/nushell#5621] [Q&A]
Nushell only seems to prompt its internal fucntion when pressing `tab`. I wonder if there is way to make it prompt the excutables on PATH also.

Also I am using nushell from Windows.

**Comment by kubouch** [comment]
I think it can be done but it's probably not implemented yet. Can you make a feature request issue for it (if one does not exist yet)?

**Reply by NoahELE** [reply]
Sure.
